### PR TITLE
Simplify GranitProtocolDecoder command result detection

### PR DESCRIPTION
**Pull request by Abyss777**
I've used the same idea with tilde and assumed that tilde has only packets with points.
If there is no tilde it is command result.
And now there is no binary to string conversion.

### CODE REVIEW

File: src/org/traccar/protocol/GranitProtocolDecoder.java [hunk 1]
```diff
@@ -132,10 +133,11 @@ private static void decodeStructure(ChannelBuffer buf, Position position) {
...
 
         ChannelBuffer buf = (ChannelBuffer) msg;
-        String bufString = buf.toString(StandardCharsets.US_ASCII);
-        if (hasDeviceId() &&  bufString.contains("OK") || bufString.startsWith("ERROR")
-                || bufString.startsWith("+PR") || bufString.startsWith("+IDNT")
-                || bufString.startsWith("+BBMD")) {
+
+        int indexTilde = buf.indexOf(buf.readerIndex(), buf.writerIndex(), new StringFinder("~"));
+
+        if (hasDeviceId() &&  indexTilde == -1) {
```
Comment (tananaev) [inline src/org/traccar/protocol/GranitProtocolDecoder.java]: Please remove double space.

**Comment by tananaev** [comment]
One tiny comment. Otherwise looks good.

**Comment by Abyss777** [comment]
Checkstyle missed it, and me too.
Done.

**Comment by tananaev** [comment]
Merged. Thanks again.

**Pull request merged**